data/helpers.d/utils

```diff
@@ -24,3 +24,19 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to simplify its call
+#
+# Helpfully mostly for curl arguments
```

Comment:
I think Bram wrote in the chatroom about that. It is simpler and more gratifying for apps packagers. We should decide about that point :-)
You can use use "deal" instead of "fiddle", it is simpler.